changes/fa0f0afe26db13ddd9d35b3d1c784801.yaml

```diff
@@ -0,0 +1,5 @@
+---
+desc: Fix an AttributeError in ``synapse.tools.changelog``
+prs: []
```

Comment:
```suggestion
prs: [3798]
```

Reply:
I want to leave this empty so I have a steel thread to implement pulling the numbers from git.